distribution/pom.xml

```diff
@@ -171,7 +171,6 @@
 
     <modules>
         <module>fully-loaded</module>
```

Comment:
I think at the very least it means we should keep something of the `smoke-test-shaded` logic that is currently planned to be removed here and call it `smoke-test-client` so that we don't have these surprises?